### ISSUE TITLE
False positive SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName

### ISSUE THREAD

**Issue by dereuromark** [slevomat/coding-standard#1265]
We would like to enable this sniff, but the side effects right now killing our code a bit too high
It seems to also remove some inside docblocks, which should not happen (they are FQCN inside for a reason), but it also seems to kill code inside strings.

This is an example patch of the changes it did to our code:

```diff
@@ -23,6 +23,7 @@
 use Spryker\Shared\Config\Config;
 use Spryker\Shared\ErrorHandler\ErrorHandlerEnvironment;
 use Spryker\Shared\PropelOrm\PropelOrmConstants;
+use Spryker\Zed\Kernel\Persistence\EntityManager\TransferToEntityMapper;
 
 class ObjectBuilder extends PropelObjectBuilder
 {
@@ -108,8 +109,7 @@
         // When this is true we will not check for value equality as we need to be able to set a value for this field
         // to it's initial value and have the column marked as modified. This is relevant for update cases when
         // we create an instance of an entity manually.
-        // @see \Spryker\Zed\Kernel\Persistence\EntityManager\TransferToEntityMapper::mapEntity()
-        \$hasDefaultValue = $hasDefaultValue;
+TransferToEntityMapper        \$hasDefaultValue = $hasDefaultValue;
 
         if (\$hasDefaultValue || \$this->$clo !== \$v) {
             \$this->$clo = \$v;
@@ -151,8 +151,7 @@
         // When this is true we will not check for value equality as we need to be able to set a value for this field
         // to it's initial value and have the column marked as modified. This is relevant for update cases when
         // we create an instance of an entity manually.
-        // @see \Spryker\Zed\Kernel\Persistence\EntityManager\TransferToEntityMapper::mapEntity()
-        \$hasDefaultValue = $hasDefaultValue;
+TransferToEntityMapper        \$hasDefaultValue = $hasDefaultValue;
 
         if (\$hasDefaultValue || \$this->$clo !== \$v) {
             \$this->$clo = \$v;

@@ -22,6 +22,7 @@
 use Spryker\Shared\Config\Config;
 use Spryker\Shared\ErrorHandler\ErrorHandlerEnvironment;
 use Spryker\Shared\PropelOrm\PropelOrmConstants;
+use Spryker\Zed\Kernel\Persistence\EntityManager\TransferToEntityMapper;
 
 class ObjectBuilderWithLogger extends PropelObjectBuilder
 {
@@ -80,8 +81,7 @@
         // When this is true we will not check for value equality as we need to be able to set a value for this field
         // to its initial value and have the column marked as modified. This is relevant for update cases when
         // we create an instance of an entity manually.
-        // @see \Spryker\Zed\Kernel\Persistence\EntityManager\TransferToEntityMapper::mapEntity()
-        \$hasDefaultValue = $hasDefaultValue;
+TransferToEntityMapper        \$hasDefaultValue = $hasDefaultValue;
 
         if (\$hasDefaultValue || \$this->$clo !== \$v) {
             \$this->$clo = \$v;

@@ -13,6 +13,7 @@
 
 use Exception;
 use Propel\Generator\Builder\Om\TableMapBuilder as PropelTableMapBuilder;
+use Locator;
 
 class TableMapBuilder extends PropelTableMapBuilder
 {
@@ -30,8 +31,7 @@
 
         return "
                 /** @var \\Generated\\Zed\\Ide\\AutoCompletion \$locator */
-                \$locator = \\Spryker\\Zed\\Kernel\\Locator::getInstance();
-                $objName = \$locator->"
+Locator                $objName = \$locator->"
                     . lcfirst($bundle)
                     . '()->entity'
                     . $this->getObjectClassName() . '();';

**Comment by kukulich** [comment]
As a hotfix you can probably use ' insteaf of "
